bin/dart_openapi_codegen.dart

```diff
@@ -633,14 +682,27 @@ class Operation {
       (path.endsWith('/') ? '/' : '') +
       "'";
 
-  String get dartQueryMap =>
-      '{\n' +
-      parameters.entries
-          .where((e) => e.value.type == ParameterType.query)
-          .map((e) =>
-              '      ${e.value.schema.dartToQueryEntry(e.key, variableName(e.key))},\n')
-          .join('') +
-      '    }';
+  String get dartQueryMap {
+    final buffer = StringBuffer('{\n');
+
+    for (final entry in parameters.entries) {
+      if (entry.value.type == ParameterType.query) {
+        switch (entry.value.explode) {
+          case true:
+            buffer.write(
+                '${entry.value.schema.dartCondition(variableName(entry.key))}  ${entry.value.schema.dartType.startsWith('List') ? '"${entry.key}": ${variableName(entry.key)}' : '...${variableName(entry.key)}'}');
+            break;
```

Comment:
Nit but:
```suggestion
          case true when entry.value.schema.dartType.startsWith('List'):
            buffer.write(
                '${entry.value.schema.dartCondition(variableName(entry.key))} "${entry.key}": ${variableName(entry.key)}');
            break;
          case true:
            buffer.write(
                '${entry.value.schema.dartCondition(variableName(entry.key))} ...${variableName(entry.key)}');
            break;
```

It took me quite a while to untangle that to parse it, I hope I did it correctly?

Reply:
oo fancy, let me test

Reply:
works